wikitemplate.md

```diff
@@ -123,7 +125,15 @@
 
 ## Geolocation
 
-- [ ] Check that https://developer.mozilla.org/en-US/docs/Web/API/Geolocation/Using_geolocation works
+- [ ] Check that https://developer.mozilla.org/en-US/docs/Web/API/Geolocation/Using_geolocation is blocked due to cross origin iframes
+- [ ] Check that https://browserleaks.com/geo works and shows correct location
+- [ ] Check that  https://html5demos.com/geo/ works but need not be accurate location
+
+## Crash Reporting
+
+- [ ] Check `chrome://crash` causes the new tab to crash 
+- [ ] Check `chrome://crashes` lists all the crash with a crash id
```

Comment:
nitpick: "Check that `chrome://crashes` lists all the crashes and includes both `Crash Report ID` & `Local Crash ID`"